xhtml2pdf/files.py

```diff
@@ -414,4 +414,6 @@ def getFile(*a, **kw):
 
 
 def cleanFiles():
```

Comment:
The original implementation of cleanFiles didn't do anything. Code `files_tmp = []` will create only local variable of the function.

It is required to close files manually and then clear the original instance of the list.